db.py

```diff
@@ -91,6 +92,36 @@ def confirm_couriering(user_pubkey, escrow_pubkey, location, kwargs=None, photo=
     add_event(user_pubkey, events.COURIER_CONFIRMED, location, escrow_pubkey, kwargs=kwargs, photo=photo)
 
 
+def send_notification(event_type, escrow_pubkey):
+    """Send notification to users."""
+    if not escrow_pubkey or event_type not in (
+            events.LAUNCHED, events.COURIER_CONFIRMED, events.COURIERED, events.RECEIVED):
+        return
+
+    package = get_package(escrow_pubkey)
+    notification_body = 'Please check your Packages archive for more details'
+    if event_type == events.LAUNCHED:
+        notifications.send_notifications(
+            get_active_tokens(package['recipient_pubkey']),
+            title="You have new package {}".format(package['short_package_id']),
+            body=notification_body)
+    elif event_type == events.COURIER_CONFIRMED:
+        notifications.send_notifications(
+            get_active_tokens(package['launcher_pubkey']),
+            title="Courier confirmed for package {}".format(package['short_package_id']),
+            body=notification_body)
+    elif event_type == events.COURIERED:
+        notifications.send_notifications(
+            get_active_tokens(package['recipient_pubkey']),
+            title="Your package {} in transit".format(package['short_package_id']),
+            body=notification_body)
+    elif event_type == events.RECEIVED:
+        notifications.send_notifications(
+            get_active_tokens(package['launcher_pubkey']),
+            title="Your package {} delivered".format(package['short_package_id']),
+            body=notification_body)
+
+
```

Comment:
@yaroslavr-itd , can we send some kind of code, so the actual message will be set by the mobile app (to include translations)?

Reply:
Sure!
But need to coordinate this with @DenisDemyanko

Reply:
Added in [2727584](https://github.com/paket-core/router/commit/2727584c1c123be78a577ad75352fac4995111bd)